pages/common/javac.md

```diff
@@ -5,3 +5,19 @@
 - Compile a .java file:
 
 `javac {{filename.java}}`
+
+- Compile two or more .java files:
+
+`javac {{filename1.java}} {{filename2.java}} {{filename3.java}}`
+
+- Compile all .java files in current directory:
+
+`javac {{*.java}}`
+
+- Compile a .java file and place class file in a specific directory:
+
+`javac -d {{path/to/some/directory}} {{filename.java}}`
+
+- Compile two or more .java files and place class files in a specific directory:
```

Comment:
Yeah, I don't think that's asking too much. At best we could provide an example combining two options (where at least one of them is new), but at some point this ceases being a primer to the specific command and becomes a primer to the command line, so we have to draw the line somewhere. Feel free to make a judgment call :)